cmd/cmd.go

```diff
@@ -365,9 +384,41 @@ func (o *options) generateTemplate() (*template.Template, error) {
 			}
 			return strings.TrimSuffix(string(b), "\n"), nil
 		},
+		"formatTsRFC3339Nano": func(ts any) string {
```

Comment:
I think the `Ts` part does not give additional information for users, so what do you think about simply naming it `rfc3339Nano` or `toRFC3339Nano`?

```
{{ colorGreen (rfc3339Nano $msg.ts) }}
```

Reply:
Sounds good to me, I don't have a strong opinion here 👍

Reply:
Thanks for changing! Could you add the functions such as `toRFC3339Nano` to the table in the templates section?


<img width="823" alt="image" src="https://user-images.githubusercontent.com/9250296/222857273-600df816-6159-4d03-bc1f-41b50b3eeb93.png">

Reply:
👍 adding all of them them